### DISCUSSION TITLE
Error while installing dompdf with all dependencies

### DISCUSSION THREAD

**Discussion by ballotade** [dompdf/dompdf#2964] [Q&A]
Hello,
i used the composer.json file here in github (https://github.com/dompdf/dompdf/blob/master/composer.json) to install dompdf with all the dependencies and there was the following error:
_In ClassMapGenerator.php line 74:
Could not scan for classes inside "lib/" which does not appear to be a file nor a folder_

I'm not very common with installing php plugins with composer, so please help me. :)
Greetings,
Fab

**Comment by bsweeney** [comment]
To install Dompdf as a dependency for your own project you don't use the composer.json from our project. Instead you create your own composer.json then specify Dompdf as a dependency.

composer.json
```
{
    "require": {
        "dompdf/dompdf"
    }
}
```

Then run `composer update` to actually install Dompdf.

Or, as indicated in the [README](https://github.com/dompdf/dompdf#install-with-composer) you can run `composer require dompdf/dompdf`.

Finally, just reference the Composer autoloader in your own PHP to make Dompdf available within your project.

```
require 'vendor/autoload.php';
```

The [Composer documentation](https://getcomposer.org/doc/01-basic-usage.md) is a good general resource for getting started with Composer.

**Reply by ballotade** [reply]
Oh, is it that easy? Thank you very much! :)
